orchestra/tests/test_dashboard.py

```diff
@@ -470,11 +472,12 @@ def test_assignment_snapshot_upgrades(self):
         good_snapshots = {
             'snapshots': [{
                 'data': {},
-                'datetime': datetime.utcnow().isoformat(),
+                'datetime': (
+                    datetime.utcnow().replace(tzinfo=tzutc()).isoformat()),
```

Comment:
Can we use something from `django.util.timezone` here? I get nervous about doing timezones right in python...